src/keywords.js

```diff
@@ -10,6 +10,13 @@ export syntax galat = function (ctx) {
     return #`false`;
 }
 
+
+// isNaN
+export syntax kya_galat = function (ctx) {
+    let params = ctx.next().value
+    return #`isNaN ${params}`;
+}
+
 // if
```

Comment:
It will be good if we can have list of possible suggestions because pronunciation and writing in present case can cause confusion.
btw is there any such pure urduish for such a case.
or can be like `kyaNumber` **or** `kyanumber` `numberHae` 
like in English we have a short form for `NaN` i.e. **Not-a-Number**, can't  it could be `KnH` **Kya-number-Hae** or something similar ?